### PR TITLE
sentry errors

### PR DESCRIPTION
**Pull request by ice-orion**

### CODE REVIEW

**Review by ice-atlas — APPROVED** [review]

**Review by ice-ares — APPROVED** [review]

File: src/store/rootSaga.ts [hunk 1]
```diff
@@ -60,3 +65,28 @@ export function* rootSaga(): SagaIterator {
   yield cancel(spawnedSagas);
   yield call(rootSaga);
 }
+
+const isImportantError = (error: unknown) => {
```
Comment (ice-atlas) [inline src/store/rootSaga.ts:69]: `isImportantError` is not really clear name for it. Maybe smth like `isNeedToLogError`. Also nice to add `/** description */` for it :)

File: src/store/modules/Account/sagas/verifyBeforeUpdateEmail.ts [hunk 2]
```diff
@@ -68,9 +74,22 @@ export function* verifyBeforeUpdateEmailSaga({
...
+        case ValidateError.SameEmail:
+          validateError = t('errors.same_email');
+          break;
+      }
+    }
+
     yield put(
       AccountActions.VERIFY_BEFORE_UPDATE_EMAIL.FAILED.create(
-        getAuthErrorMessage(error) ?? getErrorMessage(error),
+        validateError ?? getAuthErrorMessage(error) ?? getErrorMessage(error),
```
Comment (ice-atlas) [inline src/store/modules/Account/sagas/verifyBeforeUpdateEmail.ts:92]: May we encapsulate `getAuthErrorMessage` in `getErrorMessage`?

**Review by ice-atlas — CHANGES_REQUESTED** [review]

File: src/store/rootSaga.ts (hunk 1 above)
Comment (ice-ares) [reply src/store/rootSaga.ts:69]: IsUnexpected

File: src/utils/phoneNumber.ts [hunk 3]
```diff
@@ -42,28 +42,10 @@ export const formatPhoneNumber = (
  * countryCode helps to parse numbers in national format
  * e.g. 8 (909) 999-66-99 -> +79099996699
  */
-export const e164PhoneNumber = (
```
Comment (ice-ares) [inline src/utils/phoneNumber.ts:45]: Why? U still need e164 for firebase as well

**Review by ice-ares — DISMISSED** [review]

File: src/utils/phoneNumber.ts (hunk 3 above)
Comment (ice-orion) [reply src/utils/phoneNumber.ts:45]: i actually removed `beautifyPhoneNumber` method -> git diff shows it weird and `e164PhoneNumber` is now below this change

File: src/store/rootSaga.ts (hunk 1 above)
Comment (ice-orion) [reply src/store/rootSaga.ts:69]: done

**Review by ice-ares — APPROVED** [review]

**Review by ice-atlas — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**